main.go

```diff
@@ -187,6 +187,19 @@ func toolVersion(tool string) (string, error) {
 	return out, nil
 }
 
+func find(root, ext string) ([]string, error) {
+	var matches []string
+	if walkErr := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() && filepath.Ext(path) == "."+ext {
```

Comment:
using `"."+ext` makes impossible to check files without extension(the hardcoded "." keeps blocking because `filepath.Ext(path)` returns empty string if no `.` found reading from backwards.

for example
```
filepath.Ext("") == "."+"" -> "" != "."
```

Not sure if required to check files without extension